packages/ddp-client/common/namespace.js

```diff
@@ -13,25 +13,28 @@ LivedataTest.SUPPORTED_DDP_VERSIONS = DDPCommon.SUPPORTED_DDP_VERSIONS;
 // This is private but it's used in a few places. accounts-base uses
 // it to get the current user. Meteor.setTimeout and friends clear
 // it. We can probably find a better way to factor this.
-DDP._CurrentMethodInvocation = new Meteor.EnvironmentVariable;
-DDP._CurrentPublicationInvocation = new Meteor.EnvironmentVariable;
+DDP._CurrentMethodInvocation = new Meteor.EnvironmentVariable();
+DDP._CurrentPublicationInvocation = new Meteor.EnvironmentVariable();
 
 // XXX: Keep DDP._CurrentInvocation for backwards-compatibility.
 DDP._CurrentInvocation = DDP._CurrentMethodInvocation;
 
-DDP.ConnectionError = Meteor.makeErrorType(
-  "DDP.ConnectionError", function (message) {
-    var self = this;
-    self.message = message;
+DDP.ConnectionError = Meteor.makeErrorType('DDP.ConnectionError', function(
+  message
+) {
```

Comment:
I find it quite interesting that `prettier` would wrap long lines in this manner (by splitting the `function` arguments across lines) when it opted for the following style (splitting on the right-hand side of an assignment) in code [further down in this review](https://github.com/meteor/meteor/pull/9337/files#diff-5fd7a402a518dc533b2f1c08652dbc57R43):

```js
      self.statusListeners =
        typeof Tracker !== 'undefined' && new Tracker.Dependency();
```
🤷‍♂️